src/topology/algebra/group.lean

```diff
@@ -401,6 +406,40 @@ begin
   exact hf'.dense_image hf hs
 end
 
+/-- The topological closure of a normal subgroup is normal.-/
+@[to_additive "The topological closure of a normal additive subgroup is normal."]
+lemma subgroup.is_normal_topological_closure {G : Type*} [topological_space G] [group G]
+  [topological_group G] (N : subgroup G) [N.normal] :
+  (subgroup.topological_closure N).normal :=
+{ conj_mem := λ n hn g,
+  begin
+    apply mem_closure_of_continuous (topological_group.continuous_conj g) hn,
+    intros m hm,
+    exact subset_closure (subgroup.normal.conj_mem infer_instance m hm g),
+  end }
+
+/-- The connected component of 1 is a subgroup of `G`. -/
+@[to_additive "The connected component of 0 is a subgroup of `G`."]
+def subgroup.connected_component_of_one (G : Type*) [topological_space G] [group G]
+  [topological_group G] : subgroup G :=
+{ carrier  := connected_component (1 : G),
+  one_mem' := mem_connected_component,
+  mul_mem' := λ g h hg hh,
+  begin
+    rw connected_component_eq hg,
+    have hmul: g ∈ connected_component (g*h),
+    { apply continuous.image_connected_component_subset (continuous_mul_left g),
+      rw ← connected_component_eq hh,
+      exact ⟨(1 : G), mem_connected_component, by simp only [mul_one]⟩, },
+    simpa [← connected_component_eq hmul] using (mem_connected_component)
+  end,
+  inv_mem' := λ g hg,
+  begin
+    rw ← one_inv,
+    exact continuous.image_connected_component_subset continuous_inv _
+      ((set.mem_image _ _ _).mp ⟨g, hg, rfl⟩)
+  end }
```

Comment:
Similarly, this could be a `inv_mem_connected_component_one` lemma

Reply:
I wrote this one too, but I think it still needs the `[topological_group G]` hypothesis.